include/deal.II/particles/particle_handler.h

```diff
@@ -830,10 +830,10 @@ namespace Particles
     types::particle_index global_number_of_particles;
 
     /**
-     * This variable stores how many particles are stored locally. It is
+     * This variable stores how many particles are locally owned. It is
      * calculated by update_cached_numbers().
      */
-    types::particle_index local_number_of_particles;
+    types::particle_index locally_owned_particles;
```

Comment:
I like having something like "n_" or "number" in the name to indicate it is a count instead of a container of objects.

Reply:
I agree, but the function is already called n_locally_owned_particles. Should I rename to `number_of_locally_owned_particles`?

Reply:
Yes :-)

Reply:
I should have guessed the answer to that one :-)